tests/app/views/contexts/test_thank_you_context.py

```diff
@@ -0,0 +1,69 @@
+from datetime import datetime, timezone
+from functools import cache
+
+import pytest
+from flask import Flask
+
+from app.data_models.session_data import SessionData
+from app.views.contexts.thank_you_context import build_thank_you_context
+
+SURVEY_TYPE_DEFAULT = "default"
+SURVEY_TYPE_SOCIAL = "social"
+SUBMITTED_AT = datetime.now(timezone.utc)
+
+
+@pytest.fixture
+def fake_session_data():
+    return SessionData(
+        tx_id="tx_id",
+        schema_name="some_schema_name",
+        language_code=None,
+        launch_language_code=None,
+        survey_url=None,
+        ru_ref="ru_ref",
+        response_id="response_id",
+        case_id="case_id",
+        period_str=None,
+        ru_name=None,
+    )
+
+
+def test_context_correct_if_survey_type_social(fake_session_data, app: Flask):
```

Comment:
Probably don't need the correct_if in the name as we don't do that anywhere else. I would probably do something like test_social_suvery_context

Reply:
Removed "correct_if" and renamed so it's the one you recommend now, thanks!